### PR TITLE
#159965260 Setup Travis, Code Climate and Heroku for deployment

### PR DESCRIPTION
**Pull request by pngg**
#### What does this PR do?
Update TravisCI to run tests and run Code Climate test coverage, deploy app to Heroku

#### Description of Task to be completed
- Update TravisCI to run tests
- Run codeclimate test coverage
- Setup app for Heroku auto deployment

#### How should this be manually tested?
- Navigate to the following URL
[`Frontend React Application`](https://magnificent6-frontend.herokuapp.com/)

#### What are the relevant pivotal tracker stories?
[Update TravisCI to run tests](https://www.pivotaltracker.com/story/show/159965260)

[Run codeclimate test coverage](https://www.pivotaltracker.com/story/show/159965263)

#### Screenshots
screen_shot_2018-10-04_at_10 06 42

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - npm run build
+after_script:
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+notifications:
+  email: false
+  slack: andela:a9Wc1dsD5EuUXLwDC2DtqUDh
+deploy:
+  provider: heroku
+  app:
+    develop: magnificent6-frontend
```
Comment (micahoriaso) [inline .travis.yml:25]: Add an empty last line here

**Review by micahoriaso — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (pngg) [reply .travis.yml:25]: @micahoriaso there's a newline character there. Github removes it to shorten the file. The original file has it.

Comment (micahoriaso) [reply .travis.yml:25]: Okay

**Comment by micahoriaso** [comment]
This chore works properly

**Review by micahoriaso — APPROVED** [review]

**Comment by mikenthiwa** [comment]
Looks good. Sorry about the late reply

**Pull request merged**